Utilities/src/test/java/edu/stanford/hivdb/mutations/MutationTest.java

```diff
@@ -538,4 +540,100 @@ public void testGenePosition() {
 		assertEquals(mutRT67.getGenePosition(), new GenePosition(Gene.RT, 67));
 		assertEquals(mutIN155.getGenePosition(), new GenePosition(Gene.IN, 155));
 	}
+	
+	@Test
+	public void testIsDeletion() {
+		final Mutation del = new Mutation(Gene.PR, 68, "-");
+		final Mutation ins = new Mutation(Gene.PR, 68, "_");
+		final Mutation mut = new Mutation(Gene.PR, 68, "N");
+		final Mutation delMut = new Mutation(Gene.PR, 68, "N-");
+		final Mutation delIns = new Mutation(Gene.RT, 67, "_-");
+		assertTrue(del.isDeletion());
+		assertFalse(ins.isDeletion());
+		assertFalse(mut.isDeletion());
+		assertFalse(delMut.isDeletion());
+		assertFalse(delIns.isDeletion());
+	}
+	
+	@Test
+	public void testHasStop() {
+		final Mutation mut = new Mutation(Gene.PR, 68, "N");
+		final Mutation stop = new Mutation(Gene.RT, 67, "*");
+		final Mutation stopMut = new Mutation(Gene.IN, 155, "N*");
+		assertFalse(mut.hasStop());
+		assertTrue(stop.hasStop());
+		assertTrue(stopMut.hasStop());
+	}
+	
+	@Test
+	public void testIsUnusual() {
+		final Mutation unusualMut = new Mutation(Gene.RT, 1, "A");
+		final Mutation usualMut = new Mutation(Gene.RT, 1, "H");
+		final Mutation usualMuts = new Mutation(Gene.RT, 1, "HLPST");
+		final Mutation unusualMuts = new Mutation(Gene.RT, 1, "ACDEFG");
+		final Mutation mixedMuts = new Mutation(Gene.PR, 75, "AILMVSTY");
+		assertFalse(usualMut.isUnusual());
+		assertFalse(usualMuts.isUnusual());
+		assertTrue(unusualMut.isUnusual());
+		assertTrue(unusualMuts.isUnusual());
+		assertTrue(mixedMuts.isUnusual());
+			
+//		The JSON suggests so, but are some insertions and deletions 'usual?' 
```

Comment:
Yes. It's possible to have some intertions and deletions to be usual.

Reply:
Gotcha. Thanks for clarifying. I've removed my question from the code.